torchgeo/datasets/cropharvest.py

```diff
@@ -0,0 +1,317 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""CropHarvest datasets."""
+
+import glob
+import json
+import os
+from typing import Callable, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import torch
+from matplotlib.figure import Figure
+from torch import Tensor
+
+from .geo import NonGeoDataset
+from .utils import download_and_extract_archive, download_url
+
+
+class CropHarvest(NonGeoDataset):
+    """CropHarvest dataset.
+
+    The CropHarvest <https://github.com/nasaharvest/cropharvest> datataset is a
+    classification dataset.
+
+    Dataset features:
+
+    * _ pixel single timeseries with croptype labels
+    * 18 bands per image over 12 months
+
+    Dataset format:
+
+    * images are 12x18 ndarrays with 18 bands over 12 months
+
+    Dataset properties:
+
+    1. is_crop - whether or not a single pixel contains cropland
+    2. classification_label - optional field identifying a specific croptype
+    3. dataset - source datset for the imagery
```

Comment:
```suggestion
    3. dataset - source dataset for the imagery
```
So different predictions were made by different satellites?

Reply:
I believe this field just refers to where the labels come from while imagery is standardized and downloaded from eo_data according to the cropharvest docs.